services/crc/index.html

```diff
@@ -3,3 +3,22 @@
 title: Aerogear CRC
 section: mobile-services
 ---
+
+
+<h2 class="section-header"><i class="fa fa-plug" aria-hidden="true"></i>
+    Custom Runtime Connector</h2>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit ante. Proin cursus sagittis vehicula. Nulla mattis id erat non sagittis. Nullam ut ligula venenatis, ultrices nulla a, convallis sem. Interdum et malesuada fames ac ante ipsum primis in faucibus. Vestibulum tincidunt, erat quis tincidunt bibendum, est sapien viverra erat, et dictum felis dolor vel nulla. Sed hendrerit tincidunt ligula id mollis. Nam iaculis lectus risus, sed suscipit dui tempus vitae. Pellentesque nec ultrices risus, sit amet vehicula nisi. </p>
+
+
+<h3>Getting Started</h3>
+<p>This guide will show you how to set up a Custom Runtime Connector and consume it from your mobile client.
+    <a href="https://github.com/aerogearcatalog/custom-runtime-connector-apb/blob/master/docs/provision.adoc">Geting started with the Custom Runtime Connector</a>
```

Comment:
Do you think it is a good idea to set `target="_blank"` for external links so website visitors won't  leave the website?